.github/workflows/comment-on-base-branch-change.yml

```diff
@@ -0,0 +1,22 @@
+name: Post Integration Test Comment on Base Branch Change
+
+on:
+  pull_request:
+    types:
+      - edited
```

Comment:
Will we get this message everytime we make any new commit or update anything in the PR while being based on master all the time?

Reply:
So edited event is triggered on following things:
- title change
- description change
- base branch change

We will only get this message when our base branch is changed from some X branch to "master". The following check in the action is making it sure.

```
if: ${{ github.event.changes.base && github.event.pull_request.base.ref == 'master' }}
```

Reply:
By default github doesn't give any native event for base branch. We have to subscribe on "edited" and cover base change through if condition.